src/platform/BUILD.gn

```diff
@@ -241,21 +241,40 @@ if (chip_device_platform != "none" && chip_device_platform != "external") {
         "cc13x2_26x2/ConfigurationManagerImpl.cpp",
         "cc13x2_26x2/ConnectivityManagerImpl.cpp",
         "cc13x2_26x2/ConnectivityManagerImpl.h",
+        "cc13x2_26x2/DeviceNetworkProvisioningDelegateImpl.cpp",
+        "cc13x2_26x2/DeviceNetworkProvisioningDelegateImpl.h",
         "cc13x2_26x2/InetPlatformConfig.h",
         "cc13x2_26x2/Logging.cpp",
         "cc13x2_26x2/PlatformManagerImpl.cpp",
         "cc13x2_26x2/PlatformManagerImpl.h",
         "cc13x2_26x2/SystemPlatformConfig.h",
       ]
 
+      if (chip_enable_ble) {
+        configs -= [ "${build_root}/config/compiler:std_default" ]
+        configs += [ "//:sdk_posix_config" ]
```

Comment:
This needs fixing. I think you just need to  add `"${sdk_target_name}_posix_config"` to **public_configs** of `"${chip_root}/third_party/ti_simplelink_sdk"`  rather than **configs**.